wolfcrypt/src/wc_pkcs11.c

```diff
@@ -747,9 +784,10 @@ static int Pkcs11CreateRsaPrivateKey(CK_OBJECT_HANDLE* privateKey,
         { CKA_EXPONENT_2,       NULL,          0                    },
         { CKA_COEFFICIENT,      NULL,          0                    },
         { CKA_PUBLIC_EXPONENT,  NULL,          0                    },
-        { CKA_ID,               NULL,          0                    }
+        { 0,                    NULL,          0                    },
+        { 0,                    NULL,          0                    }
     };
-    CK_ULONG        keyTmplCnt = sizeof(keyTemplate) / sizeof(*keyTemplate) - 1;
+    CK_ULONG        keyTmplCnt = sizeof(keyTemplate) / sizeof(*keyTemplate) - 2;
```

Comment:
Can you clarify why the empty entries above are required and the `-2`?

Reply:
Comments added.